lib/improver/utilities/save.py

```diff
@@ -33,6 +33,36 @@
 import iris
 
 
+def append_metadata_cube(cubelist):
```

Comment:
This function needs some sort of unit tests. You could check that the resulting prefix cube actually has the attributes that you're adding, as well as that the resulting cube has a `bald__isPrefixedBy` attribute.

Reply:
New test class and unittests added to check prefix cube attributes and bald__isPrefixedBy attribute  on other cubes in cube list.